website/siteConfig.js

```diff
@@ -4,6 +4,15 @@
 const repoUrl = 'https://github.com/scalameta/scalafmt';
 const baseUrl = '/scalafmt/';
 
+const users = [
+  {
+    caption: 'buildo',
+    image: 'https://www.buildo.io/static/media/logo.60e4e7fc.svg',
+    infoLink: 'https://www.buildo.io',
+    pinned: true,
```

Comment:
`pinned` controls whether a user ends up on the homepage. It probably makes sense to have some famous projects/companies here.